explainability-service/src/main/java/org/kie/trustyai/service/endpoints/health/ServiceHealthCheck.java

```diff
@@ -0,0 +1,17 @@
+package org.kie.trustyai.service.endpoints.health;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.eclipse.microprofile.health.HealthCheck;
+import org.eclipse.microprofile.health.HealthCheckResponse;
+import org.eclipse.microprofile.health.Startup;
+
+@Startup
+@ApplicationScoped
+public class ServiceHealthCheck implements HealthCheck {
+
+    @Override
+    public HealthCheckResponse call() {
+        return HealthCheckResponse.up("Service startup health");
```

Comment:
Are you sure this is necessary?
I expect this behaviour is OOTB adding the extension

Reply:
@danielezonca, good point. It's not necessary, I'll remove it.